api/controllers/link-controller.ts

```diff
@@ -120,7 +118,7 @@ export const fetchLink = async (shortCode: string) => {
  */
 export const fetchMyLinks = async (linkIds: string[]) => {
   try {
-    const result = await DatabaseService.getInstance()
+    let result = await DatabaseService.getInstance()
```

Comment:
why let?

Reply:
Oh I was testing some stuff and forgot to make it back to const